src/winforms/toga_winforms/app.py

```diff
@@ -30,14 +30,14 @@ def create(self):
             # SetProcessDpiAwareness(True)
             if ((win_version.Major == 6 and win_version.Minor == 3) or
                     (win_version.Major == 10 and win_version.Build < 15063)):
-                shcore.SetProcessDpiAwareness(True)
+                ctypes.windll.shcore.SetProcessDpiAwareness(True)
```

Comment:
The preferred approach here would be to assume that you can always import the `shcore` symbol from `.libs`, but it might be `None` in some situations. This is because the dll lookup is slow, so we only want to do it once. We can then put an if check around any usage.

In this case, since we know `shcore` won't exist on a particular Windows version range, we can ensure that we never use the shcore symbol with that version check.

Reply:
ok, good to know that dll lookup may be slow. New ver pushed